client/ayon_core/pipeline/create/context.py

```diff
@@ -1049,6 +1054,35 @@ def add_publish_attr_defs_change_callback(
             PUBLISH_ATTR_DEFS_CHANGED_TOPIC, callback
         )
 
+    def add_instance_requirement_change_callback(
+        self, callback: Callable
+    ) -> "EventCallback":
+        """Register callback to listen instance requirement changes.
+
+        Create plugin changed attribute definitions of instance.
+
+        Data structure of event::
```

Comment:
Is the double `::` intentional?

```suggestion
        Data structure of event:
```